yarn.lock

```diff
@@ -12426,7 +12426,7 @@ uuid@^3.0.1, uuid@^3.3.2:
     eslint-plugin-react "^7.12.4"
     eslint-plugin-react-hooks "^1.6.0"
     eslint-plugin-scanjs-rules "^0.1.4"
-    eslint-plugin-va-enzyme "file:../../Library/Caches/Yarn/v4/npm-vagov-eslint-1.0.0-4e4c7dca-961b-4a86-a8d1-85502669dab7-1555430367307/node_modules/vagov-eslint/custom-eslint/va-enzyme"
+    eslint-plugin-va-enzyme "file:../../Library/Caches/Yarn/v4/npm-vagov-eslint-1.0.0-502d90db-d4ff-4ec6-9676-2b8486c70c63-1556302093236/node_modules/vagov-eslint/custom-eslint/va-enzyme"
```

Comment:
weird, but ok

Reply:
Yeah, this is an artifact of how we're copying around linting configs. I'd like to see if we can get rid of it eventually, but @rianfowler knows more about it.

Reply:
I think that's the idiomatic way unless we want to publish it to npm